app/helpers/helpers/helpers_file.go

```diff
@@ -26,6 +59,24 @@ func ParsePackage(text string) string {
 	return text[start:stop]
 }
 
+func GetFileName(path string) string {
+	var res string
+	var found bool
+
+	for ix, character := range path {
```

Comment:
use strings package  
https://golang.google.cn/src/strings/strings.go?s=1920:1956#L66

Reply:
addressed in https://github.com/TudorHulban/GoLayouter/commit/d9c081de52e70f86c4ee0af1d15f3af69343fabf